extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/impl/PropertyRules.java

```diff
@@ -44,22 +46,26 @@ public PropertyRules inclusive(String one, String other) {
     }
 
     public void check(Properties properties) {
+        List<String> errors = new ArrayList<>();
         for (String mandatory : required) {
             if (!properties.containsKey(mandatory)) {
-                throw new IllegalStateException(mandatory + " must be specified");
+               errors.add(mandatory + " must be specified");
             }
         }
 
         for (Map.Entry<String, String> entry : excludes.entrySet()) {
             if (properties.containsKey(entry.getKey()) && properties.containsKey(entry.getValue())) {
-                throw new IllegalStateException(entry.getKey() + " and " + entry.getValue() + " are mutually exclusive");
+                errors.add(entry.getKey() + " and " + entry.getValue() + " are mutually exclusive");
             }
         }
 
         for (Map.Entry<String, String> entry : includes.entrySet()) {
             if (properties.contains(entry.getKey()) && !properties.contains(entry.getValue())) {
-                throw new IllegalArgumentException(entry.getKey() + " requires " + entry.getValue() + " to be set too");
+                errors.add(entry.getKey() + " requires " + entry.getValue() + " to be set too");
             }
         }
+        if (!errors.isEmpty()) {
+            throw new IllegalStateException(String.join(", ", errors));
+        }
     }
```

Comment:
Let's add simple unit test for this.

Reply:
Good idea, it turned out there was a typo :D